app/models/pglogical_subscription.rb

```diff
@@ -1,36 +1,99 @@
 require 'pg/dsn_parser'
 require 'pg/logical_replication'
+require 'query_relation'
+
+class PglogicalSubscription
+  include Vmdb::Logging
+  extend QueryRelation::Queryable
+
+  attr_accessor :id, :status, :dbname, :host, :user, :password, :port, :provider_region, :provider_region_name
+
+  # Create and return a new PglogicalSubscription instance. The possible
+  # options that can be passed to the constructor are:
+  #
+  # * id
+  # * status
+  # * dbname
+  # * host
+  # * user
+  # * password
+  # * port
+  # * provider_region
+  # * provider_region_name
+  #
+  def initialize(**kwargs)
+    kwargs.each do |key, value|
+      raise ArgumentError, "invalid key '#{key}'" unless attributes.include?(key)
+
+      send("#{key}=", value)
+    end
+  end
 
-class PglogicalSubscription < ActsAsArModel
-  set_columns_hash(
-    :id                   => :string,
-    :status               => :string,
-    :dbname               => :string,
-    :host                 => :string,
-    :user                 => :string,
-    :password             => :string,
-    :port                 => :integer,
-    :provider_region      => :integer,
-    :provider_region_name => :string
-  )
-
-  def self.find(*args)
-    case args.first
-    when :all then find_all
-    when :first, :last then find_one(args.first)
-    else find_id(args.first)
+  # A list of attributes typically used on inspection, and necessary
+  # for ActiveModel::AttributeMethods, if used.
```

Comment:
Since we're not using ActiveModel I think this comment needs updating, right?

Reply:
Yep, updated.